test/clojure/core_test/add_watch.cljc

```diff
@@ -17,7 +17,7 @@
                       (let [do-update (fn [x]
                                         (try
                                           (swap! x inc)
-                                          (catch #?(:cljs :default :clj clojure.lang.ExceptionInfo) e
+                                          (catch #?(:cljs :default :default clojure.lang.ExceptionInfo) e
```

Comment:
For every other `Exception` type, we introduced `:cljr`, but for these three in this file, we use `:default`. Let's please use `:cljr`. I'm hoping to phase out all `:default` branches of reader conditionals, since interop can really have no cross-dialect default.

Reply:
Yes, I can do these while merging. Will do tomorrow.